mule-deployer/src/main/java/org/mule/tools/model/anypoint/Cloudhub2DeploymentSettings.java

```diff
@@ -13,17 +13,29 @@
  */
 public class Cloudhub2DeploymentSettings extends RuntimeFabricDeploymentSettings {
```

Comment:
It extends RuntimeFabricDeploymentSettings and not need to define a new HTTP.
![image](https://github.com/mulesoft/mule-maven-plugin/assets/33416070/8709a47a-5470-4d55-9390-ea91b8c6ca09)
The CH 2.0 deployment config works just fine as described:
```
<plugin>
  <groupId>org.mule.tools.maven</groupId>
  <artifactId>mule-maven-plugin</artifactId>
  <version>3.7.1</version>
  <extensions>true</extensions>
  <configuration>
    <cloudhub2Deployment>
      <uri>https://anypoint.mulesoft.com</uri>
      <provider>MC</provider>
      <environment>${environment}</environment>
      <target>${targetName}</target>
      <muleVersion>${muleVersion}</muleVersion>
      <username>${user}</username>
      <password>${pass}</password>
      <applicationName>${appName}</applicationName>
      <replicas>1</replicas>
      <vCores>1</vCores>
      <deploymentSettings>
        <http>
          <inbound>
            <publicUrl>${publicURL}</publicUrl>
            <pathRewrite>${pathRewrite}</publicUrl>
            <forwardSslSession>true</forwardSslSession>
            <lastMileSecurity>true</lastMileSecurity>
          </inbound>
        </http>
      </deploymentSettings>
    </cloudhub2Deployment>
  </configuration>
</plugin>
```
Also for the **pathRewrite** that is not as well documented here (https://docs.mulesoft.com/mule-runtime/latest/deploy-to-cloudhub-2#cloudhub2-deploy-reference), it works when you place it under deploymentSettings -> http -> inbound.